terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -1 +1,43 @@
-# placeholder file
\ No newline at end of file
+aws_bucket = "staging-eu-west-2.terraform-state.ch.gov.uk"
+remote_state_bucket = "ch-service-staging-terraform-state"
+environment = "staging"
+deploy_to = "staging"
+state_prefix = "env:/staging"
+aws_profile = "staging-eu-west-2"
+kms_alias = "alias/staging-eu-west-2/environment-services-kms"
+
+# scaling configs
+desired_task_count = 2 # use multi instance in staging
```

Comment:
@tobafolaranmi-ch we think this is needed for Staging. Could you please confirm.

Reply:
Yes it is needed as it's staging